hooks/useConversation.ts

```diff
@@ -51,24 +52,53 @@ const useConversation = (
       }
       setLoading(false)
     }
+    listMessages()
+  }, [conversation, convoMessages])
+
+  useEffect(() => {
+    if (!conversation) return
     const streamMessages = async () => {
-      const stream = await conversation.streamMessages()
-      setStream(stream)
+      stream = await conversation.streamMessages()
       for await (const msg of stream) {
         if (dispatchMessages) {
           await dispatchMessages({
             peerAddress: conversation.peerAddress,
             messages: [msg],
           })
         }
+        if (
+          latestMsgId !== msg.id &&
+          Notification.permission === 'granted' &&
+          msg.senderAddress !== walletAddress &&
+          !browserVisible
+        ) {
+          new Notification('New Message On XMTP', {
```

Comment:
I think "New Message" might be redundant here.

```suggestion
          new Notification('XMTP', {
```